fastlane/metadata/android/pl-PL/title.txt

```diff
@@ -1 +1 @@
-Element
+Element - Bezpieczny komunikator
```

Comment:
this will cause the release to fail due to the length being more than 30 characters